test/SwooleRequestHandlerRunnerFactoryTest.php

```diff
@@ -122,9 +124,9 @@ public function testFactoryWillUseConfiguredStaticResourceHandlerWhenPresent()
             ->get(StaticResourceHandlerInterface::class)
             ->will([$this->staticResourceHandler, 'reveal']);
 
-        $factory = new SwooleRequestHandlerRunnerFactory();
-        $runner = $factory($this->container->reveal());
-        $this->assertInstanceOf(SwooleRequestHandlerRunner::class, $runner);
-        $this->assertAttributeSame($this->staticResourceHandler->reveal(), 'staticResourceHandler', $runner);
+            $factory = new SwooleRequestHandlerRunnerFactory();
+            $runner = $factory($this->container->reveal());
+            $this->assertInstanceOf(SwooleRequestHandlerRunner::class, $runner);
+            $this->assertAttributeSame($this->staticResourceHandler->reveal(), 'staticResourceHandler', $runner);
```

Comment:
and here

Reply:
I'd previously wrapped those in a Swoole\Process before realising I could use ->createMock() on the Server - Prophecy can't mock Swoole Server FWIW. I missed undoing the indentation when I removed the changes!